datasets/modeled/transport/dimensions/enduses.csv

```diff
@@ -0,0 +1,12 @@
+id,name,fuel_id,unit
+electricity_ev_ldv_home_l1,ev_ldv_home_l1,electricity,kWh
+electricity_ev_ldv_home_l2,ev_ldv_home_l2,electricity,kWh
+electricity_ev_ldv_work_l1,ev_ldv_work_l1,electricity,kWh
+electricity_ev_ldv_work_l2,ev_ldv_work_l2,electricity,kWh
+electricity_ev_ldv_public_l2,ev_ldv_public_l2,electricity,kWh
+electricity_ev_ldv_public_dcfc,ev_ldv_public_dcfc,electricity,kWh
+electricity_ev_ldv_enroute_dcfc,ev_ldv_enroute_dcfc,electricity,kWh
+electricity_ev_mhdv_depot_ac,ev_mhdv_depot_ac,electricity,kWh
+electricity_ev_mhdv_depot_dc,ev_mhdv_depot_dc,electricity,kWh
+electricity_ev_mhdv_opportunity_dc,ev_mhdv_opportunity_dc,electricity,kWh
+electricity_ev_mhdv_enroute_dc,ev_mhdv_enroute_dc,electricity,kWh
```

Comment:
The dataset has rows with `electricity_rail_transit`.

Reply:
oops... thanks, fixed